libs/ngx-dsfr/button/src/lib/testing/button.component.spec.ts

```diff
@@ -30,7 +28,7 @@ describe('DsfrButtonComponent', () => {
   let dsfrButtonHarness: DsfrButtonHarness;
 
   const testLabel = 'Test Label';
-  const testTitle = 'Test Title';
+  const testIcon = 'fr-icon-checkbox-circle-line'
```

Comment:
`const testIcon = 'checkbox-circle-line';`

At this level, should be just the name of the icon, not the CSS class that the DSFR expects.